test/util.js

```diff
@@ -58,18 +57,27 @@ exports.addUserWithNGroups = function (nGroups, username = 'user', groupName = '
   });
 };
 
-exports.addUserWithGroup = function (username = 'user', groupName = 'Group') {
-  return exports.addUserWithNGroups(1, username, groupName).then((res) => {
+exports.addUserWithGroup = function (name = 'User', groupName = 'Group') {
+  return exports.addUserWithNGroups(1, name, groupName).then((res) => {
     return { user: res.user, group: res.groups[0] };
   });
 };
 
-exports.addUser = function (username = 'user') {
-  return exports.addUserWithNGroups(0, username).then((res) => {
+exports.addUser = function (name = 'User') {
+  return exports.addUserWithNGroups(0, name).then((res) => {
     return res.user;
   });
 };
 
+exports.addNUsers = function (nUsers, name = 'User') {
```

Comment:
I added this and then realized what I actually needed was addNArticles lol... I'll leave it though even though it's unused.